qgisfeedproject/templates/layouts/base-fullscreen.html

```diff
@@ -50,12 +50,15 @@
       <div id="menu" class="navbar-menu">
           <div class="navbar-end">
               
-                  
-                  
-                  <a class="navbar-item has-text-weight-bold is-size-7 is-active" href="/accounts/login">
-                      <span>Login</span>
-                  </a>
-                  
+            {% if user.is_authenticated and user.is_staff %}
```

Comment:
@Xpirix, why can only staff logout?

Reply:
@dimasciput , As in the administration page, only superuser (`is_superuser=True`, `is_staff=True`) or staff(`is_superuser=False`, `is_staff=True`) users can log in to the management page. However, there is the default user called 'qgis_user' that is authenticated automatically but is not an admin or staff. Thus, only authenticated admin and staff users can log in and see the logout button.

![image](https://github.com/qgis/qgis-feed/assets/43842786/e40c5900-134a-4f79-8b65-bea571d6903f)

Reply:
@dimasciput , I think this needs to be fixed in the future beacause if we remove the condition `user.is_staff`, we will always see the logout button even if there is no authenticated user.

Reply:
Ah ok, yes we can re-visit this later.